libyul/YulStack.cpp

```diff
@@ -205,18 +194,6 @@ MachineAssemblyObject YulStack::assemble(Machine _machine) const
 	{
 	case Machine::EVM:
```

Comment:
I think we should get rid of the enum altogether, but maybe in a second PR.

Reply:
I just created #14009 that is removing the enum.